.gitignore

```diff
@@ -1,6 +1,7 @@
 # IDE
 .idea
-.vscode
+.vscode.
+.env
```

Comment:
oh, I see you added it. then you need to delete it, make a commit and the add it again locally